src/Entities/Page/PageBuilder.cs

```diff
@@ -544,36 +544,37 @@ public PageBuilder WithAttachmentsFactory(Func<ValueTask<IEnumerable<FileAttachm
     /// <inheritdoc/>
     IPage IPageBuilder<IPage>.Build() => Build();
 
-    internal PageBuilder WithPaginatorFooter(PaginatorFooter footer, int page, int totalPages, ICollection<IUser>? users)
+    internal PageBuilder WithPaginatorFooter(PaginatorFooter footer, int currentPageIndex, int maxPageIndex, ICollection<IUser>? users)
     {
         if (footer == PaginatorFooter.None)
-        {
             return this;
-        }
 
         Footer = new EmbedFooterBuilder();
+
         if (footer.HasFlag(PaginatorFooter.Users))
         {
-            if (users is null || users.Count == 0)
+            if (users?.Any() != true)
             {
-                Footer.Text += "Interactors: Everyone\n";
+                Footer.Text += "Interactors: Everyone";
             }
             else if (users.Count == 1)
             {
-                var user = users.First();
+                var user = users.Single();
 
-                Footer.IconUrl = user.GetAvatarUrl() ?? user.GetDefaultAvatarUrl();
-                Footer.Text += $"Interactor: {user}\n";
+                Footer.Text += $"Interactor: {user}";
+                Footer.IconUrl = user.GetDisplayAvatarUrl();
             }
             else
             {
                 Footer.Text += $"Interactors: {string.Join(", ", users)}";
             }
+
+            Footer.Text += Environment.NewLine;
```

Comment:
Discord uses line feed characters for new lines, so it's not necessary to use `Environment.NewLine`.

```suggestion
            Footer.Text += '\n';
```